dyno-queues-redis/src/main/java/com/netflix/dyno/queues/demo/DynoQueueDemo.java

```diff
@@ -56,6 +74,40 @@ public static void main(String[] args) throws IOException {
         }
     }
 
+    private void runSimpleV1Demo(DynoJedisClient dyno) throws IOException {
+        String region = System.getProperty("LOCAL_DATACENTER");
+        String localRack = System.getProperty("LOCAL_RACK");
+
+        String prefix = "dynoQueue_";
+
+        DynoShardSupplier ss = new DynoShardSupplier(dyno.getConnPool().getConfiguration().getHostSupplier(), region, localRack);
+
+        RedisQueues queues = new RedisQueues(dyno, dyno, prefix, ss, 50_000, 50_000);
+
+        Message msg1 = new Message("id1", "searchable payload");
+        Message msg2 = new Message("id2", "payload 2");
+        Message msg3 = new Message("id2", "payload 3");
+        DynoQueue V1Queue = queues.get("simpleQueue");
+
+        // Test push() API
+        //List pushed_msgs = V1Queue.push(Arrays.asList(msg1));
+        List pushed_msgs = V1Queue.push(ImmutableList.of(msg1, msg2, msg3));
+
+        // Test ensure() API
+        System.out.println("Does Message with ID '" + msg1.getId() + "' already exist? " + !V1Queue.ensure(msg1));
```

Comment:
<nit> maybe move to a logger if possible

Reply:
Good catch. Done in the next revision.